src/app/pages/jails/jail-list/jail-list.component.ts

```diff
@@ -132,17 +132,20 @@ export class JailListComponent implements OnInit {
   constructor(protected router: Router, protected rest: RestService, protected ws: WebSocketService, 
     protected loader: AppLoaderService, protected dialogService: DialogService, private translate: TranslateService) {}
 
-  public tooltipMsg: any = T("Choose an existing ZFS Pool to allow the iocage jail manager \
-  to create a /iocage dataset in the selected pool. The '/iocage' dataset may not be visible \
-  until after the first jail is created. iocage uses this dataset to store FreeBSD RELEASES \
-  and all other jail data. To create a new ZFS Pool, navigate Storage/Volumes and click 'Create ZFS Pool'.");
+  public tooltipMsg: any = T("Choose an existing ZFS Pool to allow the \
```

Comment:
"Choose a pool where the iocage jail manager can create the /iocage dataset. The /iocage..."